MantidPlot/src/main.cpp

```diff
@@ -154,6 +154,11 @@ the indentation depth for him/herself.
 */
 
 int main(int argc, char **argv) {
+  // Force the qtpy package to use the PyQt4 bindings. QtPy < v1.4 does not
+  // check whether a particular binding is imported and selects PyQt5 by default
+  // causing MantidPlot to segfault.
+  qputenv("QT_API", "pyqt");
```

Comment:
Please remove it from `buildconfig/CMake/Packaging/launch_mantidplot.sh.in` so it isn't set in two places

Reply:
:+1: